_posts/release/2017-09-05-accumulo-2.0.0.md

```diff
@@ -18,10 +18,22 @@ always return `false` instead of `true`, even if the contents are equal.
 [ACCUMULO-3652] - Replaced string concatenation in log statements with slf4j
 where applicable. Removed tserver TLevel logging class.
 
+[ACCUMULO-4733] - Introduced a new configuration option,
+crypto.security.provider, to allow for a configurable security provider for
+crypto. If unset, it will default to the system level Provider list.
+
+[ACCUMULO-4737] - Renamed configuration option crypto.cipher.algorithm.name to
+crypto.cipher.key.algorithm.name. The option now functions as described in
+Property.java. Introduced a new configuration option, crypto.wal.cipher.suite,
```

Comment:
The documentation in Property.java will eventually end up in the user manual.  So could say as described in the user manual.  Also should be able to link to it eventually.